WalletWasabi/WabiSabi/Backend/Rounds/Round.cs

```diff
@@ -123,5 +126,35 @@ public bool IsInputRegistrationEnded(uint maxInputCount, TimeSpan inputRegistrat
 
 			return false;
 		}
+
+		private uint256 CalculateHash()
+		{
+			var strobe = new Strobe128("WabiSabi_v1.0");
+
+			void append(string fieldName, object fieldValue)
```

Comment:
I don't like to receive `object` because it assumes all the receive instances override the `object.ToString()` method. This means that if I delete one of those `ToString` methods everything will compile just okay. This scenario is not crazy because the IDEs are not able to detect that the overriden `ToString` methods are used. Honestly I don't understand why something like a `CredentialIssuerParameters` has a `ToString`.

Also, using the `ToString` is not ideal because it serializes the internal state and then we use that string to get the `byte[]`. I think the easiest alternative would be to overload the `append` method to receive `ulong` and `CredentialIssuerParameters`.